Description: Specify release as bookworm rather than trixie.
 A couple of scripts and config files used during the installation process
 hardcode "13" or "Trixie" as the Debian version. This is inaccurate and
 result in either visual issues or system issues if not fixed. This patch
 fixes those.
Author: Aaron Rainbolt <arraybolt3@ubuntu.com>
Origin: Debian
Last-Update: 2024-07-25

---

--- calamares-settings-debian-13.0.6.orig/scripts/sources-final
+++ calamares-settings-debian-13.0.6/scripts/sources-final
@@ -4,7 +4,7 @@
 #

 CHROOT=$(mount | grep proc | grep calamares | awk '{print $3}' | sed -e "s#/proc##g")
-RELEASE="trixie"
+RELEASE="bookworm"

 cat << EOF > $CHROOT/etc/apt/sources.list
 # See https://wiki.debian.org/SourcesList for more information.
--- calamares-settings-debian-13.0.6.orig/scripts/sources-media
+++ calamares-settings-debian-13.0.6/scripts/sources-media
@@ -2,7 +2,7 @@

 CHROOT=$(mount | grep proc | grep calamares | awk '{print $3}' | sed -e "s#/proc##g")
 MEDIUM_PATH="/run/live/medium"
-RELEASE="trixie"
+RELEASE="bookworm"

 if [ "$1" = "-u" ]; then
     umount $CHROOT/$MEDIUM_PATH
--- calamares-settings-debian-13.0.6.orig/calamares/branding/debian/branding.desc
+++ calamares-settings-debian-13.0.6/calamares/branding/debian/branding.desc
@@ -9,10 +9,10 @@ windowPlacement: center
 strings:
     productName:         Debian GNU/Linux
     shortProductName:    Debian
-    version:             13
-    shortVersion:        13
-    versionedName:       Debian 13
-    shortVersionedName:  Debian 13
+    version:             12
+    shortVersion:        12
+    versionedName:       Debian 12
+    shortVersionedName:  Debian 12
     bootloaderEntryName: Debian
     productUrl:          https://debian.org
     supportUrl:          https://www.debian.org/support
